flow/react-navigation.js

```diff
@@ -144,9 +146,37 @@ declare module 'react-navigation' {
     uri: string,
   |};
 
+  declare export type NavigationReplaceAction = {|
+    +type: 'Navigation/REPLACE',
```

Comment:
As far as I understand, there are no reasons to make any of these properties covariant. Can you remove the `+` signs please?

Reply:
Here's a good primer on property variance: https://flow.org/blog/2016/10/04/Property-Variance/

Reply:
Hmm.. why not? We used this all over the place at FB to make properties read-only

It would be pretty crazy if users were allowed to do `action.type = 'Overridden'`